qiita_marker.gemspec

```diff
@@ -2,28 +2,28 @@
 
 lib = File.expand_path('lib', __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'commonmarker/version'
+require 'qiita_marker/version'
 
 Gem::Specification.new do |s|
-  s.name = 'commonmarker'
-  s.version = CommonMarker::VERSION
+  s.name = 'qiita_marker'
+  s.version = QiitaMarker::VERSION
   s.summary = 'CommonMark parser and renderer. Written in C, wrapped in Ruby.'
   s.description = 'A fast, safe, extensible parser for CommonMark. This wraps the official libcmark library.'
   s.authors = ['Garen Torikian', 'Ashe Connor']
-  s.homepage = 'https://github.com/gjtorikian/commonmarker'
+  s.homepage = 'https://github.com/gjtorikian/qiita_marker'
```

Comment:
This url is wrong. The correct one is here.
https://github.com/increments/qiita_marker

Reply:
@kyntk 
This pull request only do the mechanical renaming.
I'll fix the `gemspec` in another pull request.